src/LiveComponent/assets/src/Backend/BackendResponse.ts

```diff
@@ -13,4 +13,8 @@ export default class {
 
         return this.body;
     }
+
+    async getBlob(): Promise<Blob> {
+        return await this.response.blob();
```

Comment:
Prefer `return` instead of `return await`: 
```suggestion
        return this.response.blob();
```